audit/test/audit_test.cpp

```diff
@@ -22,3 +22,47 @@ TEST(audit_test, no_errors_initially){
     EXPECT_EQ(a.error_count(), 0u);
     EXPECT_EQ(a.error_strings().size(), 0u);
 }
+
+TEST(audit_test, conflicting_leaders){
```

Comment:
brace should be on a new line.
Also, the test name should be a bit more descriptive.